post_request_task/task.py

```diff
@@ -5,7 +5,7 @@
 from django.core.signals import (got_request_exception, request_finished,
                                  request_started)
 
-from celery import task as base_task
+from celery import shared_task as base_task
```

Comment:
What prompted this change ?

Reply:
With Celery 5 we will get ModuleNotFoundError: No module named 'celery.task'. Task module is deprecated in Celery 5. We can use either the @app.task or the @shared_task decorator. We can use shared_task in both 4 and 5 versions.